website/src/components/SearchPage/Table.tsx

```diff
@@ -134,106 +136,114 @@ export const Table: FC<TableProps> = ({
         );
 
     return (
-        <div className='w-full overflow-x-auto text-sm' aria-label='Search Results Table'>
-            <Tooltip id='table-tip' />
-            {data.length !== 0 ? (
-                <table className='min-w-full text-left border-collapse'>
-                    <thead>
-                        <tr className='border-gray-400 border-b mb-100'>
-                            <th className='px-2 py-2 md:pl-6 text-xs text-gray-500 cursor-pointer text-left'>
-                                {selectedSeqs.size > 0 && (
-                                    <MaterialSymbolsClose className='inline w-3 h-3 mx-0.5' onClick={clearSelection} />
-                                )}
-                            </th>
-                            <th
-                                onClick={() => handleSort(primaryKey)}
-                                className='px-2 py-2 md:pl-6 text-xs font-medium tracking-wider text-gray-500 uppercase cursor-pointer text-left'
-                            >
-                                {capitalCase(primaryKey)} {orderBy.field === primaryKey && orderIcon}
-                            </th>
-                            {columns.map((c) => (
+        <div className='text-sm'>
```

Comment:
note to reviewers - the only actual change here is wrapping stuff in ScrollContainer - the diffs are just from the change to the indentation that results